style.css

```diff
@@ -4,4 +4,31 @@ main {
 
 aside {
```

Comment:
will be good if you will add * { margin: 0; padding: 0 } for removing default spaces from browser
or just add reset.css link to html